### ISSUE TITLE
Dashboard is cached in chrome

### ISSUE THREAD

**Issue by godstanis** [containous/traefik#6514]
This issue could be a known one or resolved in the next versions, but I don't currently have time to test it properly, sorry if this one's fixed. I will try to look deeper into this issue once I have more time.

### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

When traefik service is fully disabled I open  `http://traefik.localhost/dashboard/#/` page in chrome

### What did you expect to see?

Nothing/blank not resolved page/not working page.

### What did you see instead?

Everything is not working, and all other routes are not found, like `traefik.localhost` and `traefik.localhost/dashboard`, traefik container and services are 100% down. But this particular one is cached:

![image](https://user-images.githubusercontent.com/19244786/76967577-af65f080-6938-11ea-8d31-46cf74a6fd57.png)

I guess it's somehow cached in chrome because in firefox or in incognito chrome this page is not working. I also can't get rid of it.

I tried emptying cache and hard reload but this page still is showing. Even if I set v1 traefik with the same domain - this page still shows and messes up the old one.

### Output of `traefik version`: (_What version of Traefik are you using?_)
`v2.0.7` (i use `traefik:v2.0` image from docker hub)

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
# Docker configuration backend
providers:
  docker: {}

# Traefik dashboard
api:
  dashboard: true
  insecure: true
```

Docker-compose:
```
  # Traefik
  proxy:
    image: traefik:v2.0
    volumes:
      - ./traefik.yml:/etc/traefik/traefik.yml
      - /var/run/docker.sock:/var/run/docker.sock
    ports:
      - "80:80"
      - "8080:8080"
    labels:
      - "traefik.http.routers.api.rule=Host(`traefik.localhost`)"
      - "traefik.http.routers.api.service=api@internal"
```

**Comment by SantoDE** [comment]
Hello @godstanis,

our Dashboard is delivered as a "PWA" (Progress Web App). For that reason, the actual dashboard  code (js, css..) is stored within the browsers local storage in order to speed up the rendering process.

For that reason, you can see the dashboard even tough the API is not answering and not providing you with data to show.

Does it make sense?

Therefore, I'll close that issue as it's not a bug but rather, a performance improvement.

**Comment by mhilker** [comment]
The issue does also occur when traefik is running fine but the dashboard is protected with basic auth. The browser popup is not displayed on page load when the pwa is cached. You have to clear your cache or open the developer console with `disable cache` and reload the page. This forces the popup to be shown.

Maybe something like [this](https://thatemil.com/blog/2018/02/21/pwa-basic-auth/) would be necessary?

**Comment by traefiker** [comment]
Closed by #6717.
